addons/stock_delivery/tests/test_packing_delivery.py

```diff
@@ -78,7 +78,7 @@ def test_put_in_pack_weight_wizard(self):
         # We instanciate the wizard with the context of the action and check that the
         # default weight was set.
         pack_wiz = self.env['choose.delivery.package'].with_context(pack_action_ctx).create({})
-        self.assertEqual(pack_wiz.shipping_weight, 13.5)
+        self.assertEqual(pack_wiz.shipping_weight, 12)
```

Comment:
Since you fixed this, you can also give a purpose of this wizard :relieved: (see the test explanation :sweat_smile: )